### DISCUSSION TITLE
localized text feature implementation

### DISCUSSION THREAD

**Discussion by developera** [FreeOpcUa/opcua-asyncio#889] [Ideas]
Hi,

There is any plan to implement "localized text feature"?

I need this feature and it should be easy to implement. As workaround I overrided line 452 params.LocaleIds.append("en") in client.py. Not sure if there is any other solution for the moment.

Thanks and regards

**Comment by oroulet** [comment]
I do not have any plans and I have no ideas how that stuff should be used

**Comment by schroeder-** [comment]
See #890
